src/core/census/grpc_context.c

```diff
@@ -39,7 +39,7 @@ static void grpc_census_context_destroy(void *context) {
 }
 
 void grpc_census_call_set_context(grpc_call *call, census_context *context) {
-  if (!census_available()) {
+  if (census_enabled() == CENSUS_FEATURE_NONE) {
```

Comment:
Not sure I understand that comment - census_available() is killed in this PR. Do you mean something with census_supported()?